src/app/dev/DevToys.Tools/Tools/Converters/JsonTable/JsonTableConverter.resx

```diff
@@ -0,0 +1,172 @@
+              </xsd:sequence>
+              <xsd:attribute name="name" use="required" type="xsd:string" />
+              <xsd:attribute name="type" type="xsd:string" />
+              <xsd:attribute name="mimetype" type="xsd:string" />
+              <xsd:attribute ref="xml:space" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="assembly">
+            <xsd:complexType>
+              <xsd:attribute name="alias" type="xsd:string" />
+              <xsd:attribute name="name" type="xsd:string" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="data">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+                <xsd:element name="comment" type="xsd:string" minOccurs="0" msdata:Ordinal="2" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" msdata:Ordinal="1" />
+              <xsd:attribute name="type" type="xsd:string" msdata:Ordinal="3" />
+              <xsd:attribute name="mimetype" type="xsd:string" msdata:Ordinal="4" />
+              <xsd:attribute ref="xml:space" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="resheader">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" />
+            </xsd:complexType>
+          </xsd:element>
+        </xsd:choice>
+      </xsd:complexType>
+    </xsd:element>
+  </xsd:schema>
+  <resheader name="resmimetype">
+    <value>text/microsoft-resx</value>
+  </resheader>
+  <resheader name="version">
+    <value>2.0</value>
+  </resheader>
+  <resheader name="reader">
+    <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <resheader name="writer">
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <data name="AccessibleName" xml:space="preserve">
+    <value>JSON list to tabular form converter tool</value>
+  </data>
+  <data name="LongDisplayTitle" xml:space="preserve">
+    <value>JSON List to Table</value>
+    <comment>Grid view title</comment>
+  </data>
+  <data name="Description" xml:space="preserve">
+    <value>Convert a JSON list to tabular format, export to CSV or Excel</value>
+    <comment>Grid view description</comment>
+  </data>
+  <data name="ShortDisplayTitle" xml:space="preserve">
+    <value>JSON &gt; Table</value>
+    <comment>Sidebar heading</comment>
+  </data>
+  <data name="SearchKeywords" xml:space="preserve">
+    <value>JSON,CSV,TSV,French,Excel</value>
```

Comment:
How about adding `Tabular`, `Data` and removing `Excel`?

Reply:
I put in "Excel" because all 3 of the linked ideas/requests for this tool mention Excel. There's no special handling for Excel and I'm removing references to it from the other text resources. Should "Excel" remain in the search keywords though?

Reply:
That's a good remark. Let's keep Excel here then.